src/main/python/org/broadinstitute/hellbender/tools/gcnv/gcnvkernel/models/theano_hmm.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
```

Comment:
See #4043.  Let's also add some tests for the HMM.

Reply:
Edited your issue.